domains/POAS/events/metric/examples/example.json

```diff
@@ -0,0 +1,12 @@
+{
+  "metric": {
+    "Project": "MRLPA",
+    "Category": "metric",
+    "Subcategory": "FunnelCompletionRate",
+    "Environment": "1186",
```

Comment:
Where's this number coming from?

Reply:
it's the name of the env I'm using on mrlpa

Reply:
I'll change it to demo

Reply:
thanks

Reply:
Cool, I wondered if it was some other kind of opaque ID so wanted to check.